sound/soc/sof/topology.c

```diff
@@ -2711,6 +2732,11 @@ static int sof_link_load(struct snd_soc_component *scomp, int index,
 	if (!link->no_pcm) {
 		link->nonatomic = true;
 
+		if (link->stream_name &&
+		    (!strcmp(link->stream_name, "vm_fe_playback") ||
+		     !strcmp(link->stream_name, "vm_fe_capture")))
+			link->no_pcm = true;
+
```

Comment:
@lyakh its very hard to guess what is happening here. Can you please add a comment? 
Typically, DAI links for FE's are created by the soc core while parsing pcm elems in the topology during topology parsing . So their names are just dummy names. When do you set the stream_name's get set to vm_fe_playback/capture?

Reply:
@ranj063 I hard-code those names in the topology for now. We do need to come up with a good consistent scheme for handling of those names / VirtIO detection, etc.